config/software/libyaml.rb

```diff
@@ -30,7 +30,7 @@
 
 build do
   license "MIT"
-  license_file "./LICENSE"
+  license_file "https://raw.githubusercontent.com/yaml/libyaml/#{version}/LICENSE"
```

Comment:
Given that we do check the tar.gz sum but not the license sum, I'd argue that using the license file provided in the tar file is safer (and that's one less download). Was there a specific reason to stop using the file provided in the tar.gz?

Reply:
```suggestion
  license_file "./LICENSE"
```